publisher-template.yaml

```diff
@@ -3,9 +3,10 @@ apiVersion: v1
 metadata:
   name: aiops-publisher
 parameters:
-  - description: Kafka server and port
-    name: KAFKA_SERVER
-    required: true
```

Comment:
Looks like we are now making specifying the `KAFKA_SERVER` parameter optional.

Well, that would potentially mean initial failure on the deployment pod, something that I was looking to avoid. Reason being, the second deployment for me, would not always go through completely in OpenShift, and then I would have to kill that pod eventually.

Reply:
Yes, it's optional in both templates with a default value. That means in other words its possible to change the value, but normally it stays as the prod Kafka. I don't understand you concern.. What's wrong?

Reply:
Just to clarify - nothing wrong with the approach you have taken here.
When I wrote the above comment, I missed the fact that even if no value is specified, it will fall back to `platform-mq-cluster-kafka.platform-mq.svc:9092`

So we are good here!